test/gtest/ucp/test_ucp_am.cc

```diff
@@ -575,6 +557,30 @@ class test_ucp_am_nbx : public test_ucp_am_base {
         return status;
     }
 
+    ucs_status_t am_data_handler_common(const void *header,
+                                        size_t header_length,
+                                        void *data, size_t length,
+                                        const ucp_am_recv_param_t *rx_param)
+    {
+
+        check_header(header, header_length);
+
+        bool has_reply_ep = get_send_flag();
+
+        EXPECT_EQ(has_reply_ep, rx_param->recv_attr &
```

Comment:
maybe need !! on 2nd param

Reply:
I think you are right. it's working because the rhs and lhs has the same numeric values, but if the rhs had a different numeric value then {0,1} you will get a wrong assertion.

You want me to fix this in this PR? It is not part of the PR.